src/pages/explore/dfi/_components/DFIStatisticsDisplay.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { SupplyData } from "@defichain/whale-api-client/dist/api/stats";
+import { StatisticPanel } from "@components/commons/StatisticPanel";
+import { Container } from "@components/commons/Container";
+import { useUnitSuffix } from "@hooks/useUnitSuffix";
+import { useTranslation } from "next-i18next";
+import { useWhaleApiClient } from "../../../../layouts/context/WhaleContext";
+import { calculatePercentage } from "../../../../shared/calculatePercentage";
+
+export function DFIStatisticsDisplay() {
+  const api = useWhaleApiClient();
+  const [supply, setSupply] = useState<SupplyData>();
+  const { t } = useTranslation("page-explore-dfi");
+
+  const { value, suffix } = useUnitSuffix(supply?.max, 1);
```

Comment:
add some context by renaming `value` and `suffix`, on first glance value and suffix can be anything
```suggestion
  const { value, suffix } = useUnitSuffix(supply?.max, 1);
```